### DISCUSSION TITLE
tout this as a MouseZoom Replacement?

### DISCUSSION THREAD

**Discussion by GitHubUnique** [linearmouse/linearmouse#142] [General]
I had to upgrade to Monterey (ugh), and in doing so can no longer use MouseZoom (unidentified developer error  benh57.com). 

Ben Hines' utility has been a lifesaver when trying to use dual 30" monitors!

Fingers crossed the LinearMouse fills this void. 

Anyway, it'd be great if you'd "MouseZoom replacement, works on Monterey, Ventura, etc." to the description of this great project so that others can find it! (it was quite hard to find LinearMouse) 

THANKS for creating / maintaining / sharing this!

**Comment by lujjjh** [comment]
Thank you for letting me know about MouseZoom. It's the first time I've heard it. I believe this thread will help MouseZoom users find LinearMouse.

Actually, I googled “MouseZoom replacement” and discovered that someone has already made a video. :)

<img width="400" alt="image" src="https://user-images.githubusercontent.com/3000535/173195239-7a83280d-817a-475b-8a4d-c80581bd71c6.png">

**Reply by GitHubUnique** [reply]
Yes, the video is how I "found you."  Installing LinearMouse shortly - thank you!
